src/screens/ExchangeModal.js

```diff
@@ -362,8 +363,9 @@ export default function ExchangeModal({
       });
     }
 
+    const outputInUSD = outputPriceValue * outputAmount;
     const gasPrice = selectedGasPrice?.txFee?.native?.value?.amount;
-    const cancelTransaction = await checkGasvInput(gasPrice, amountInUSD);
+    const cancelTransaction = await checkGasvInput(gasPrice, outputInUSD);
```

Comment:
also can we rename the param in `checkGasvInput` as it currently has the second argument labeled as `inputPrice` - can be renamed to `outputAmount`